pyproject.toml

```diff
@@ -22,10 +22,10 @@ license = {file = "LICENSE"}
 readme = "README.md"
 requires-python = ">=3.9"
 dependencies = [
-  "numpy >= 1.23.0",
-  "pandas >= 1.4.0",
-  "pyarrow >= 9.0.0",
-  "statsmodels >= 0.13.0"
+  "numpy>=1.23.1",
+  "pandas>=1.4.3",
+  "pyarrow>=9.0.0",
+  "statsmodels>=0.13.2"
```

Comment:
I bumped some of these package versions based on the [Athena Spark packages list](https://docs.aws.amazon.com/athena/latest/ug/notebooks-spark-preinstalled-python-libraries.html).